solutions/src/main/kotlin/de/thermondo/solutions/basics/Basics06Solution.kt

```diff
@@ -0,0 +1,14 @@
+package de.thermondo.solutions.basics
+
+/**
+ * Create a class named Basics6Solution
+ *
+ * Define member variables:
+ * name: String
+ * age: Int
+ * and return the data of the two people while writing unit tests.
+ */
+class Basics06Solution(name: String, age: Int) {
+    val name = name // The variables that are part of the class are called as properties.
```

Comment:
```suggestion
    val name = name
```
I would ditch this comment. It's unclear and a bit contradictory if you compare it with the problem statement description.

These are member variables within the scope of this class but are accessible (as properties) from other classes.